index.html

```diff
@@ -1,233 +1,118 @@
AAAoCN+ZPsNAAAAfSa8AQAAWLgS2Dy8Bjc7miMAAAAAAB1l+w0AAAC9JbwBAACYn63rVhuxDQAAAAAAfXNFDXB+7+QBAADoC+ENAADA5pXYZo8a3OxvXgAAAAAA9FjZfvOVJN/0TQAAAEAfCG8AAABmN9hs8yibbQAAAAAA4D5+neSzdQsOAAAAdJbwBgAA4B471s02YhsAAAAAAJibb9UNOOvMCwAAgC4S3gAAAH23f5Ld63abVX0fBgAAAAAALMD6uv3mu4YHAABA1whvAACAvtm6hjb71+02S3wHAAAAAADASFyR5HNJfm2cAAAAdIXwBgAA6IPdhzbbrHHiAAAAAAAwVv9RA5x1xgwAAEDbCW8AAIAu2rFGNuXxMCcMAAAAAAATtz7Jt5J81egBAABoM+ENAADQBSW02W0otlniVAEAAAAAoBFuTvL5JD9yHAAAALSR8AYAAGgjoQ0AAAAAALTLlUnOSnKFcwMAAKBNhDcAAEAbDCKbNUIbAAAAAABote/VAOf3jhEAAIA2EN4AwMJsnWTV0L+zbF5YOuM/abdN/CfvuIiLxm/ezIfQV87yezPvGjXbvwagKbYeCmzWbOb5FAAAAAAAaKcS4Hw+yTrnBwAAQJMJbwDgHsMxze716/DvLSaWabLhkGf417+uH3Kvr78GGIelQ5HNbh1+rgUAAAAAAO6r/CzyX+tDgAMAAEAjCW8A6JPBxd2DmGbmVzbvN7PEOIPfW1d/DbAxg+fbQWCzxvMvAAAAAAAwFOCcI8ABAACgaYQ3AHTRbkMXdw82J+zmpCdqeHPOlfXr8O/9xgfm0Hm7zRLa2GQDAAAAAABsigAHmLY19cauMy32pq4bu07C9RMAAC0gvAGgzQbba9bUi7pX2ZzQOuuHtuT8vsY5mfHB0pUd+HNClw0/Fw822Kxx4gAAAAAAwCKsr/GNAAdYqKVDP7dcVX+mOfPXxa4Nm/Bv6nNgNnIdxbqh6ywAAJgQ4Q0AbTG4oHvH+rVpH3wwfnOJdNwJBsZj6VBUs7XYEQAAAAAAmBABDjCbwc8tlw5tqBlENuXrkp5M7ar69cqhr8IcAIAxEN4A0ERLa1gz2GQjsmEhrt3I1pzhu8MIdeAeq4bWo68a+oDaczAAAAAAADBtgwDnXD/fg94Y/nnlcGCz2rfAnNxcH1fO+DUAAAsgvAGgCdbMiGxsUGAahkOdm+tWnczYtBMfRNFiM8MaH04DAAAAAABtdEGSr9Wf6QHttnTompEl9bqRVa4bGaub6zUQv6kbc9ywFABgDoQ3AEzDqvphyX41tOnLil+65+YZH+hfNfTr4XgnM8IeGLXBB9HDK9QHm2r6tEodAAAAAADojxLgfG/Gz+iA5tq13ihw9YzYhum7uV7T8Jt6M1LPqwAAMwhvAJiEpUORjY02cJfhD6pmbtXJLJt1hDv9MnMLzW7165KhsMadngAAAAAAAO76udsgwgGmb/Czzt3qzzZX+7lmK107FOFc5XoFAKDvhDcAjEv58GTfGtysNmUYudlinZkbeDb2e3GHmomYGc9kaAvNbP93ubvTVh34cwMAAAAAAEzDLTW+Oc8F4jAxg5+J7jq00cbPPLvp2qEIR4gDAPSO8AaAUVlaP0QZxDbWAUM7bWqzzua27swWA23OxsKgUZotgNmUzW2SmRnPZAH/HQAAAAAAAIzPBfXhZnQwWrvWG7EOvops+qtcP3BJkksXcJ0AAEDrCG8AWIylQ6HNviYJAAAAAAAAQIOUC8PPrReG284A87NqKLLZ1Y0I2YT1NcK5yvMtANBVwhsA5mvJUGgjtgEAAAAAAACg6dbXi8HPrTEOcF+rZ2y0sc2Ghbq0PkqIc7MpAgBdILwBYC7ENgAAAAAAAAB0wS1DW3BcEE6frR7aZrNrvTYERq3Ejhd4zgUA2k54A8CmlMjmEUn2MSUAAAAAAAAAOuayJJckudDB0gPDoc0uQhum4GdDEc56BwAAtInwBoCZVtfY5mAfsgAAAAAAAADQA+vrheCDB3TBqhrY7Ce0oYEuG4pwAAAaT3gDQOqHKwfX4GYHEwEAAAAAAACgpwYRzmUuCKdlltRtNvvUr1s5QFpgfd06Vh7XOjAAoKmENwD9tksNbg7u+yAAAAAAAAAAYAYRDk1XrvvYt4Y2brRK212X5Lz6fLveaQIATSK8AeifJfVDlye5uwkAAAAAAAAAzMn6oQDnZy4KZ0p2mfGALho83349yc1OGABoAuENQH+sqrHNPjW+AQAAAAAAAAAW5rIa4FzmwnDGSGhD35Xn2W/X51oAgKkR3gB0X/ng5TE1uAEAAAAAAAAARuuWGSEOLMRWSVYPRTY7mCLcrTzPfifJhTaOAQDTILwB6K6D6oabrZwxAAAAAAAAAEzM1TXCGTxgNrvU0Gbn+tX1HbB564cCnFvMCwCYFOENQLcsSfLYGt34QAYAAAAAAAAApk+Iw+q6wWa1bTYwMt9P8nUBDgAwCcIbgG4owc1j6mOJMwUAAAAAAACAxhqEONfVr+sdVafsUm+WusPQRhtgfC5K8u36nAoAMBbCG4B2E9wAAAAAAAAAQLvdUi8Yv7ZGOdeKcVqhxDWralizqoY2NtnA9JTnz7NtFgMAxkF4A9BOg+DmUMENAAAAAAAAAHTOIMYZBDnX1d9j8raqj12Gfm2LDTTXIMC52hkBAKMivAFoF8ENAAAAAAAAAPTX1TXAuaVudVhfoxwWZxDUrK7XY+xcv9pgA+11UQ1wRIsAwKIJbwDaYUmNbUp0s6UzAwAAAAAAAACG3DK0JWf90KYHGx/ustWMxyCqEddA95UA5xsCHABgMYQ3AM33RMENAAAAAAAAALAIgyDnlqGLz4ejnDYGOsPRzPCvB5FN6uYagNT45t/qcyEAwLwIbwCa68Aa3TzQGQEAAAAAAAAAE3L9jAvTr5vlQvX19fdHZbB9ZqaZ4cwOblwKLMKdNb75hiECAPMhvAFonp1rcLOTswEAAAAAAAAAABip/0zyl
```

Comment:
I know we will iterate and improve things as we go along, but what is there needs to work, otherwise it will confuse the user. If there is no link just yet, comment out the button for now

Reply:
Buttons will remind us to what we have to put here later on there is a chance that we might forget things

Reply:
we can raise issues to remind us, or comment it out if we will use it soon. We dont want to confuse the user or look unprofessional to potential employers or clients 😉

Reply:
Dont worry we won't forget to add the buttons. 

What happened to the buttons for Discord ...?